DEVELOPMENT-GUIDE.md

```diff
@@ -0,0 +1,85 @@
+# ACCP Development Guidance
+Correctness and code-safety is paramount in the Amazon Corretto Crypto Provider.
+
+The purpose of this guide is not to provide all information needed for development within ACCP.
```

Comment:
```suggestion
The purpose of this guide is not to provide all information needed for development within ACCP.
```
This guide is designed to ... It doesn't provide all of the information you might need ...

Reply:
As useful as this is, and I think it's very useful, I don't think this is a developer guide at all. Instead, it's a note or topic that provides guidance to developers. A developer guide would include discussion of the architecture, concepts, examples, etc.

Reply:
What should we call then then? This is as much a seed which will (hopefully) later be expanded as anything else.